src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,10 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return  Optional.ofNullable(user)
+            .map(User::getAge)
+            .filter(age -> age >= 30)
+            .filter(age -> age <= 45).isPresent();
```

Comment:
```suggestion
        return Optional.ofNullable(user)
                .map(User::getAge)
                .filter(age -> age >= 30)
                .filter(age -> age <= 45)
                .isPresent();
```
사소한 부분이지만 이번 과정을 통해서 스페이스도 신경 써보는 것은 어떨까요? 금방 손에 익으실거라 믿습니다 :)

Reply:
.으로 개행하는 부분은 연습중에 있는데 빠지는게 있네요. 네이버 코딩컨벤션은 적용시켜놓은 상태에서
줄맞춤으로 맞추고있는중인데 좀 더 신경을 쓰겠습니다.